src/services/GraphDataSource.ts

```diff
@@ -182,7 +182,7 @@ export default class GraphDataSource {
     }
 
     // Some appenders are expensive so only specify an appender if needed.
-    let appenders: AppenderString = 'deadNode,istio,healthConfig,serviceEntry,sidecarsCheck,workloadEntry';
+    let appenders: AppenderString = 'deadNode,istio,healthConfig,serviceEntry,sidecarsCheck,workloadEntry,health';
```

Comment:
Will probably change this to something like `nodeHealth` or `graphHealth` in final version

Reply:
This is likely going to change given what I did in https://github.com/israel-hdez/swscore/pull/23.  That sub-pr already introduces a new `health` appender (finalizer), and merges the healthConfig appender into it.